presto-hive/src/test/java/com/facebook/presto/hive/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -166,6 +183,60 @@ public void testFloat()
         assertEquals(getDomain(REAL, 10, floatColumnStats(maximum, minimum)), create(ValueSet.all(REAL), false));
     }
 
+    @Test
+    public void testMatchesWithStatistics()
+            throws Exception
```

Comment:
Unnecessary throws clause.

Reply:
You can simplify this method as (variables are renamed, unnecessary line breaks removed, methods static imported)

```
public void testMatchesWithStatistics()
{
    RichColumnDescriptor column = getColumn();
    String value = "Test";
    TupleDomain<ColumnDescriptor> effectivePredicate = getEffectivePredicate(column, createVarcharType(255), value);
    List<RichColumnDescriptor> columns = singletonList(column);
    TupleDomainParquetPredicate predicate = new TupleDomainParquetPredicate(effectivePredicate, columns);
    Statistics stats = getStatsBasedOnType(column.getType());
    stats.setNumNulls(1L);
    stats.setMinMaxFromBytes(value.getBytes(), value.getBytes());
    assertTrue(predicate.matches(2, singletonMap(column, stats)));
}
```